static/src/javascripts/projects/common/modules/experiments/tests/tailor-survey.js

```diff
@@ -0,0 +1,263 @@
+const inArticleWriter = (survey, surveyId) =>
+    spaceFiller.fillSpace(spacefinderRules, paras => {
+        const componentName = `data_tailor_survey_${surveyId}`;
+
+        mediator.emit('register:begin', componentName);
+        bonzo(survey).insertBefore(paras[0]);
+        mediator.emit('register:end', componentName);
+
+        return surveyId;
+    });
+
+// the main function to render the survey
+const renderQuickSurvey = () => {
+    const queryParams = {};
+
+    queryParams.edition = config.page.edition;
+
+    // If we want to force tailor to show a particular survey we can set an attribute in local storage to have
+    // key = 'surveyToShow', and value = the survey id. Tailor will then override other logic for display, and
+    // look for a survey with this ID to return. This is useful as we can easily see how a particular survey
+    // would be rendered, without actually putting it live. If this parameter is empty or not specified, tailor
+    // behaves as usual.
+    const surveyToShow = local.get('surveyToShow');
+
+    if (surveyToShow) {
+        queryParams.surveyToShow = surveyToShow;
+    }
+
+    // get the list of surveys that can't be shown as they have been shown recently
+    const surveysNotToShow = getSurveyIdsNotToShow();
+
+    if (surveysNotToShow) {
+        queryParams.surveysNotToShow = surveysNotToShow;
+    }
+
+    return tailor.getSuggestedSurvey(queryParams).then(suggestion => {
+        if (suggestion) {
+            storeSurveyShowedInCookie(suggestion.data);
+
+            const json = getJsonFromSurvey(suggestion.data.survey);
+            const survey = bonzo.create(template(tailorSurveyHtml, json));
+
+            return inArticleWriter(survey, suggestion.data.survey.surveyId);
+        }
+        Promise.resolve();
+    });
+};
+
+const disableRadioButtons = buttonClassName => {
+    const radioButtons = document.getElementsByClassName(buttonClassName);
+    bonzo(radioButtons).each(button => {
+        // eslint-disable-next-line no-param-reassign
+        button.disabled = true;
+    });
+};
+
+const surveyFadeOut = () => {
+    const surveyContent = document.getElementsByClassName(
+        'impressions-survey__content'
+    );
+
+    surveyContent[0].classList.add('js-impressions-survey__fadeout');
+};
+
+const thankyouFadeIn = () => {
+    const surveyThanks = document.getElementsByClassName(
+        'impressions-survey__thanks'
+    );
+
+    surveyThanks[0].classList.add('js-impressions-survey__fadein');
+};
+
+const recordOphanAbEvent = (answer, surveyId) => {
+    const componentId = `data_tailor_survey_${surveyId}`;
+
+    ophan.record({
+        component: componentId,
+        value: answer,
+    });
+};
+
+const handleSurveyResponse = surveyId => {
+    const surveyQuestions = [
+        ...document.getElementsByClassName('fi-survey__button'),
+    ];
+
+    surveyQuestions.forEach(question => {
+        bean.on(question, 'click', event => {
```

Comment:
given converting alone could allow errors to creep in, i think we need to be careful to limit conversion PRs to converting alone. it's a very good suggestion, but if it introduced a bug, it would be harder to pin down, given the breadth of changes already here (and that it does currently work). 

we can use `// #?` to flag things like this for review later though (even straight after merge, ideally), e.g. 
```js
       // #? use addEventListener 
       bean.on(question, 'click', event => {
```

to take it one step further, it might be a useful piece of work to remove bean as one job after this is done.